state/factory/workingset.go

```diff
@@ -689,8 +689,10 @@ func (ws *workingSet) pickAndRunActions(
 			}
 			if err := ws.txValidator.ValidateWithState(ctxWithBlockContext, nextAction); err != nil {
 				log.L().Debug("failed to ValidateWithState", zap.Uint64("height", ws.height), zap.Error(err))
-				ap.DeleteAction(nextAction.SenderAddress())
-				actionIterator.PopAccount()
+				if !errors.Is(err, action.ErrNonceTooLow) {
```

Comment:
no need to consider other errors?

Reply:
nonce too low may mean action included in parent draft block, so just ignore it. But other error is not expected